packages/components/src/InputFile/InputFile.tsx

```diff
@@ -123,7 +124,7 @@ interface InputFileProps {
    * More info is available at:
    * https://atlantis.getjobber.com/?path=/docs/components-forms-and-inputs-inputfile-docs--page#getuploadparams
    */
-  getUploadParams(file: File): UploadParams | Promise<UploadParams>;
```

Comment:
while I agree that it's odd to have a non promise here, and truthfully don't quite understand the use case for it - is it necessary to remove it as a possible type?